rpcs3/rpcs3qt/gl_gs_frame.cpp

```diff
@@ -85,9 +85,7 @@ void gl_gs_frame::delete_context(draw_context_t ctx)
 	auto gl_ctx = (GLContext*)ctx;
 	gl_ctx->handle->doneCurrent();
 
-#ifndef _WIN32
-	delete gl_ctx->handle;
-#else
+#if defined(_MSC_VER)
```

Comment:
I'd like this as `#ifdef _MSC_VER` to keep consistent with the rest of the codebase.